ui/src/app/flags/rollouts/Rollouts.tsx

```diff
@@ -137,13 +137,75 @@ export default function Rollouts(props: RolloutsProps) {
+                <p className="text-sm text-gray-500">
+                  Rules are evaluated in order from{' '}
+                  <span className="font-semibold">top to bottom</span>. The
+                  first rule that matches will be applied.
+                </p>
+                <p className="text-sm text-gray-500">
+                  <InformationCircleIcon className="mr-1 inline-block h-4 w-4 text-violet-300" />
+                  You can re-arrange rules by{' '}
+                  <span className="font-semibold">dragging and dropping</span>{' '}
+                  them into place.
+                </p>
+              </div>
+              <div
+                className="pattern-boxes w-full border p-4 pattern-bg-gray-50 pattern-gray-100 pattern-opacity-100 pattern-size-2 dark:pattern-bg-black dark:pattern-gray-900  
+  lg:w-3/4 lg:p-6"
+              >
+                {/* <DndContext
+                  sensors={sensors}
+                  collisionDetection={closestCenter}
+                  onDragStart={onDragStart}
+                  onDragEnd={onDragEnd}
+                >
+                  <SortableContext
+                    items={rules.map((rule) => rule.id)}
+                    strategy={verticalListSortingStrategy}
+                  > */}
+                <ul role="list" className="flex-col space-y-6 p-2 md:flex">
+                  {rollouts &&
+                    rollouts.length > 0 &&
+                    rollouts.map((rollout) => (
+                      <SortableRollout
+                        key={rollout.id}
+                        flagKey={flag.key}
+                        rollout={rollout}
+                        segments={segments}
+                        onQuickEditSuccess={incrementRolloutsVersion}
+                        onEdit={() => {
+                          // setEditingRollout(rollout);
+                          //setShowEditRolloutForm(true);
+                        }}
+                        onDelete={() => {
+                          setDeletingRollout(rollout);
+                          setShowDeleteRolloutModal(true);
+                        }}
+                        readOnly={readOnly}
+                      />
+                    ))}
+                </ul>
+                {/* </SortableContext>
+                  <DragOverlay>
+                    {activeRule ? (
+                      <NewRule
+                        namespace={namespace}
+                        totalRules={rules.length}
+                        rule={activeRule}
+                      />
+                    ) : null}
+                  </DragOverlay>
+                </DndContext> */}
+              </div>
+            </div>
           ) : (
             <EmptyState
               text="New Rollout"
               disabled={readOnly}
               onClick={() => {
-                setEditingRollout(null);
+                //setEditingRollout(null);
```

Comment:
this will be uncommented/added back in the next PR for allowing 'full' editing (basically to edit the description as well as the rollout fields in a traditional slideover